src/EditorFeatures/CSharpTest/Completion/CompletionProviders/KeywordCompletionProviderTests.cs

```diff
@@ -317,5 +317,19 @@ class C
 }";
             await VerifyProviderCommitAsync(markupBeforeCommit, "return", expectedCodeAfterCommit, commitChar: ';', textTypedSoFar: "return");
         }
+
+        [Fact]
+        public async Task RecommendAsyncToStartStatement()
+        {
+            var markup = @"
+class C
+{
+    public void M()
+    {
+        $$
+    }
+}";
+            await VerifyItemExistsAsync(markup, "async");
```

Comment:
Oh, yeah, I added that before I found the AsyncKeywordRecommenderTests. I'll remove it.